lib/lib-table-query/src/main/java/org/sagebionetworks/table/query/model/CurrentUserFunction.java

```diff
@@ -0,0 +1,25 @@
+package org.sagebionetworks.table.query.model;
+
+import java.util.List;
+
+public class CurrentUserFunction extends SQLElement{
```

Comment:
Most likely, since this can be used in the select you will have to implement the HasFunctionReturnType interface.